README.md

```diff
@@ -6,3 +6,79 @@ Folders in this repo:
 - Ansible: Ansible code for deploying the girder.dandiarchive.org site.
 - web: Vue.js code for holding a Girder Web Components based client deployed at gui.dandiarchive.org .
 - girder-dandi-archive: A Girder plugin for customizing the DANDI archive Girder instance.
+
+## Developing Locally
+
+### 1. Server
+
+In order to get girder up and running with the required plugins, first create a python3 virtual environment.
+
+```
+mkvirtualenv dandiarchive --python=python3
+```
+
+Install girder dandiarchive plugin in editable mode.
+
+```
+cd girder-dandi-archive
+pip install -e .
+```
+
+Build girder client.
+
+```
+girder build
+```
+
+Having a mongodb database locally is required to run Girder. If that is the case you are ready to get girder up and running.
```

Comment:
And if I don't have any?  Is there an easy way for a throw away instance (assuming I have it installed I guess) on some non privileged port?

Reply:
Again will do docker-compose which will include mongodb.